Demo/src/androidTest/java/com/braintreepayments/demo/test/CardholderNameDropInTest.java

```diff
@@ -28,7 +28,7 @@ public void setup() {
     @Test(timeout = 60000)
     public void cardholderName_whenDisabled_isHidden() {
         setCardholderNameStatus(CardForm.FIELD_DISABLED);
-        onDevice(withText("Add Payment Method")).waitForExists().waitForEnabled().perform(click());
+        onDevice(withText("Add Payment Method")).waitForExists(2000).waitForEnabled(2000).perform(click());
```

Comment:
Is this in milliseconds? If not, 2000 is long lol

Reply:
Yes this is two seconds lol, we made this change in later versions - seems to help stabilize the tests